bulk_indexer.go

```diff
@@ -160,6 +161,11 @@ func init() {
 							item.Error.Reason = ""
 						}
 
+						// For unavailable_shards_exception, remove item.Error.Reason
```

Comment:
nit: wrong comment.
nit: this if statement can be merged into the previous if statement for simplicity
```suggestion
						// For x_content_parse_exception, remove item.Error.Reason
```